arangod/IResearch/IResearchLink.cpp

```diff
@@ -609,7 +612,9 @@ IResearchLink::IResearchLink(IndexId iid, LogicalCollection& collection)
       _maintenanceState(std::make_shared<MaintenanceState>()),
       _id(iid),
       _lastCommittedTick(0),
-      _createdInRecovery(false) {
+      _createdInRecovery(false),
+      _linkStats(nullptr) {
```

Comment:
I suggest to initialize `_linkStats` with some dummy metric in order to avoid potential `nullptr` access